pallets/crowdloan/src/lib.rs

```diff
@@ -157,11 +166,21 @@ pub mod pallet {
         OptionQuery,
     >;
 
+    /// A map of crowdloan ids to their contributors count.
+    #[pallet::storage]
+    pub type ContributorsCount<T: Config> =
```

Comment:
Why did you choose a separate storage map instead of the field within the crowdload structure? We already have the "raised" computational field. The new storage map increases the DB operations number.

Reply:
At first I wanted to replace Contributions with CountedStorageMap but I decided to have a simple independent map, but you are are right it would probably make everything easier to put this in the crowdloan directly.